src/OrbitGl/App.cpp

```diff
@@ -550,6 +551,11 @@ void OrbitApp::Disassemble(int32_t pid, const FunctionInfo& function) {
   });
 }
 
+void OrbitApp::ShowSourceCode(const orbit_client_protos::FunctionInfo& function) {
+  // For now this is just a stub. This will change in a subsequent commit
```

Comment:
pers pref: //TODO(bug/feature)

Reply:
I'd agree if there were a bug which had further information about the todo. And creating a bug just for the sole purpose of having something to reference defeats the purpose.